modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormFieldTypesServlet.java

```diff
@@ -115,8 +125,27 @@ protected void doGet(
 			httpServletResponse, fieldTypesJSONArray.toJSONString());
 	}
 
+	protected Map<String, Object> getFieldProperties(
+		String ddmFormFieldName, HttpServletRequest httpServletRequest) {
+
+		if (StringUtil.equals(ddmFormFieldName, "rich_text")) {
+			EditorConfiguration richTextEditorConfiguration =
+				EditorConfigurationFactoryUtil.getEditorConfiguration(
+					StringPool.BLANK, ddmFormFieldName, "ckeditor_classic",
+					Collections.emptyMap(),
+					(ThemeDisplay)httpServletRequest.getAttribute(
+						WebKeys.THEME_DISPLAY),
+					RequestBackedPortletURLFactoryUtil.create(
+						httpServletRequest));
+
+			return richTextEditorConfiguration.getData();
+		}
+
+		return Collections.emptyMap();
```

Comment:
From @brunobasto at https://github.com/liferay-frontend/liferay-portal/pull/196#discussion_r463882448

> With this most fields will have an empty JSON object. Can ew put the check before we call this method? That way we save some data transfer.

I didn't check, so I leave that up to you. I simple removed the `JSONObject` type so it's easier to pass the `Map` around as it is serialized down the road anyways.

Reply:
@jbalsas I sent you some changes to rename `properties` in favor of `configuration` and just add `configuration` to JSON if it exists. I couldn't send it straight to your branch, I thought I had permission, I may be doing something wrong 😅. https://github.com/jbalsas/liferay-portal/pull/2225